taxi/tests/test_views.py

```diff
@@ -0,0 +1,98 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase, Client
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+MANUFACTURER_URL = reverse("taxi:manufacturer-list")
+DRIVER_URL = reverse("taxi:driver-list")
+CAR_URL = reverse("taxi:car-list")
+
+
+class PublicManufacturerTest(TestCase):
+    def test_login_required(self):
+        response = self.client.get(MANUFACTURER_URL)
+        self.assertNotEqual(200, response.status_code)
+
+
+class PrivateManufacturerTest(TestCase):
```

Comment:
Your tests are quite comprehensive, but I think It'd be better to test the search forms in the test_views.py using urls with such params. Try adding "?param=query" to your url in order to check whether search is correct)